royal-society-of-chemistry.csl

```diff
@@ -176,6 +180,30 @@
             </group>
           </group>
         </else-if>
+        <else-if type="software" match="any">
+          <group delimiter=" ">
+            <text variable="title" form="short"/>
+            <group delimiter=" " prefix="(version " suffix=")">
```

Comment:
This group as it stands currently doesn't do anything. But you're better off using `<label variable="version"/>` anyway instead of the hardcoded version in the prefix.

Reply:
Addressed the hardcoded 'version' issue.
For me, this actually works. Made sense to me to group together the 'title' and 'version' like this. But I would be OK with these being separate elements, so long as they are both present.